ddcz/templates/public.html

```diff
@@ -1,17 +1,17 @@
 {% load static %}
-<html>
```

Comment:
Maybe when fixing this, could you also add doctype? It seems I forgot this when we were doing the skin template, haven't noticed it, sorry for that.

Reply:
Right, true